1_fetch/src/get_nwis_data.R

```diff
@@ -1,8 +1,9 @@
 nwis_site_info <- function(fileout, site_hash_table){
-  site_no <- site_hash_table$filepath %>% basename() %>% 
+  site_no <- readRDS(site_hash_table) %>%
+    pull(filepath) %>% basename() %>% 
```

Comment:
I'm getting 
```
Error in basename(.) : a character vector argument expected
```
I think because `pull(filepath)` returns a vector of factors and not strings. 

fixed with 
```r
hash_df <- data.frame(filepath = names(hash_vec), hash = hash_vec, stringsAsFactors = FALSE)
```
in `hash_dir_files()`. I don't think you ran into this because that stringsAsFactors is default FALSE in R 4.0 (I am still 3.6.3).